examples/etherpad/container-src/settings.json

```diff
@@ -0,0 +1,101 @@
+{
+"title": "Etherpad",
+
+"skinName": "no-skin",
+
+"ip": "0.0.0.0",
+"port" : 9001,
+
+"showSettingsInAdminPage" : true,
+
+"dbType" : "mongodb",
+"dbSettings" : {
+                 "user"    : "DB_USERNAME",
+                 "host"    : "DB_IP",
+                 "port"    : DB_PORT,
```

Comment:
Quotes could be added around "DB_PORT".

Reply:
As this is the settings file for Etherpad, and Etherpad is picky about the database port parameter being an integer (not a string), adding the quotes would make it crash.